build_tools/github_actions/test_executable_scripts/test_hipfft.py

```diff
@@ -21,16 +21,17 @@
 # If smoke tests are enabled, we run smoke tests only.
 # Otherwise, we run the normal test suite
 test_type = os.getenv("TEST_TYPE", "full")
-if test_type == "smoke":
-    test_filter = ["--smoketest"]
-else:
-    # "--test_prob" is the probability that a given test will run.
-    # Due to the large number of tests for hipFFT, we only run a subset.
-    test_filter = [
-        "--gtest_filter=-*multi_gpu*",
-        "--test_prob",
-        "0.02",
-    ]
+if test_type != "full":
+    if test_type == "smoke":
+        test_filter = ["--smoketest"]
+    else:
+        # "--test_prob" is the probability that a given test will run.
+        # Due to the large number of tests for hipFFT, we only run a subset.
+        test_filter = [
+            "--gtest_filter=-*multi_gpu*",
+            "--test_prob",
+            "0.02",
+        ]
```

Comment:
I see python errors such as for hipfft and rocfft:

```
Traceback (most recent call last):
  File "/__w/TheRock/TheRock/build_tools/github_actions/test_executable_scripts/test_rocfft.py", line 36, in <module>
    cmd = [f"{THEROCK_BIN_DIR}/rocfft-test"] + test_filter
                                               ^^^^^^^^^^^
NameError: name 'test_filter' is not defined
```

should this just be:

```suggestion
test_filter = []
if test_type  == "smoke":
   test_filter = ["--smoketest"]
```